voltdb/README.md

```diff
@@ -0,0 +1,68 @@
+# YCSB for VoltDB
```

Comment:
needs a license header

Reply:
Should be fixed in 3eac920e3957e3b7b3190b43244bea47a4d54c64 .